app/components/margin-link.html

```diff
@@ -0,0 +1,39 @@
+<div class="container-fluid" ng-style="{
```

Comment:
The file layout in the components folder is mostly `component-name/index.ts`/`component-name/template.html`, tucking away what belongs together in one folder. Perhaps we should also apply this here.